src/constants/CardPropTypes.js

```diff
@@ -200,6 +200,19 @@ export const TableCardPropTypes = {
   ),
 };
 
+const BarChartDatasetPropType = {
```

Comment:
this is incorrect, it should have PropTypes.shape wrapped around it

Reply:
Fixed